Xpresso/model-template.yaml

```diff
@@ -32,9 +27,11 @@ dependencies:
       - numpy=1.19.2
       - pip=20.3.3
       - pysam=0.15.3
+      - cython=0.29.23
     pip:   # install via pip
       - keras==2.4.3
       - tensorflow==2.4.1
+      - git+https://github.com/kipoi/kipoiseq.git@add-xpresso-dataloader 
```

Comment:
After kipoiseq will get merged and released to pypi, then specify `kipoiseq==pypi_version`

Reply:
I have merged the pr but keeping this branch alive until the package is released. I will speak to Flo about this.